services/tenant-ui/frontend/src/assets/statusChip.scss

```diff
@@ -0,0 +1,16 @@
+// specific styling for status chips in tables
+
+.p-chip.status-chip {
+    border-radius: 5px;
+    padding: 0.25rem 1rem 0.25rem 1rem;
+    font-weight: bolder;
+  }
+
+  .p-chip.status-chip.in-progress {
+    background-color: lightsalmon;
+  }
+  
+  .p-chip.status-chip.error {
+    background-color: red;
+    color: white;
+  }
```

Comment:
Was thinking this could go in the priemeVueComponents.scss file, then wer're not importing a file per control?
Though another option that is kind of clean would be one scss file per component under a /primevueComponents folder, then import all those into the priemeVueComponents.scss, then just import that one file into the main scss?

Reply:
yeah, not really sure what you wanted to do for organizing this stuff... maybe better to just have one file for tenant-ui components (after primevue?) I figured primevue was for overriding actual primevue classes only, and then file(s )for our components that are specific to tenant ui.